06-building-a-program-start-to-finish/setting-up-continuous-integration-(build-server)/output/README.md

```diff
@@ -0,0 +1,2 @@
+#### Output
+- **TravisCI build link**: [Link to a successful TravisCI build](https://travis-ci.org/andela-oakinwa/inverted-index)
```

Comment:
Error fixed, build successful.
[Link to a successful TravisCI build](https://travis-ci.org/andela-oakinwa/inverted-index)